ui/css/module/_footnotes.scss

```diff
@@ -128,6 +128,10 @@ $footnote-vertical-margin: 24px;
     line-height: 1rem; // like font-text, except adjusting for bold
   }
 
+  a {
+    color: $color-black;
+  }
```

Comment:
Do we want this to apply to _all_ anchors at the bottom of the page, or just the markers? Designs for external links are blue, for example.

Reply:
Not at all. Good catch.